all_test.go

```diff
@@ -242,6 +244,19 @@ func TestOptions_PreserveTimes(t *testing.T) {
 	}
 }
 
+func TestOptions_CopyBufferSize(t *testing.T) {
+	opt := Options{
+		CopyBufferSize: 512,
+	}
+
+	err := Copy("test/data/case00", "test/data.copy/case00", opt)
```

Comment:
Just for naming: Why don't you append a test case, rather than using `00`?

Reply:
Oh, I didn't see that the "caseXX" files match the actual test steps. I'll change that. Thx for letting me know.

Reply:
When we add some features, it should be another test case to be appended.

Reply:
Thx. Done.